extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/devconsole/CacheDevConsoleRecorder.java

```diff
@@ -2,32 +2,73 @@
 
 import java.util.Optional;
 
+import org.jboss.logging.Logger;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
 import io.quarkus.cache.Cache;
 import io.quarkus.cache.runtime.CaffeineCacheSupplier;
 import io.quarkus.cache.runtime.caffeine.CaffeineCache;
 import io.quarkus.devconsole.runtime.spi.DevConsolePostHandler;
-import io.quarkus.devconsole.runtime.spi.FlashScopeUtil.FlashMessageStatus;
 import io.quarkus.runtime.annotations.Recorder;
+import io.quarkus.vertx.http.runtime.devmode.Json;
 import io.vertx.core.Handler;
 import io.vertx.core.MultiMap;
 import io.vertx.ext.web.RoutingContext;
 
 @Recorder
 public class CacheDevConsoleRecorder {
+    private static final Logger logger = Logger.getLogger(CacheDevConsoleRecorder.class);
 
     public Handler<RoutingContext> clearCacheHandler() {
         return new DevConsolePostHandler() {
+            int code;
+            String message = "";
+
             @Override
             protected void handlePost(RoutingContext event, MultiMap form) throws Exception {
                 String cacheName = form.get("name");
                 Optional<Cache> cache = CaffeineCacheSupplier.cacheManager().getCache(cacheName);
                 if (cache.isPresent() && cache.get() instanceof CaffeineCache) {
-                    ((CaffeineCache) cache.get()).invalidateAll();
-                    // redirect to the same page so we can make sure we see the updated results
-                    flashMessage(event, "Cache for " + cacheName + " cleared");
+                    CaffeineCache caffeineCache = (CaffeineCache) cache.get();
+
+                    String action = form.get("action");
+                    if (action.equalsIgnoreCase("clearCache")) {
+                        caffeineCache.invalidateAll();
+                        logger.info("Invalidated all element in the cache [" + cacheName + "] - cache now empty");
```

Comment:
I'm not sure this is worthy of an info level, are you?

Reply:
It's nice to see it when looking at the log (like the screenshot). But I am happy to remove it. It does give you some (more) confirmation that it happened :) w.d.y.t ?

Reply:
My reasoning is that our current policy is to log as little as possible on INFO level. Really important stuff. So is this important?

I think not, because this is for DEV mode, and the user triggers this via a click, and he gets feedback in the UI. So it serves no purpose.

Now, for tracability, we could justify this, but this would be for PROD stuff, not DEV mode.

Also, if we started logging in DEV UI actions, we'd have to do it consistently and document the policy.

So, if you have arguments that you feel outweight those considerations, let's discuss it, I'm open to change my mind :) If you're not convinced, then probably drop it down a level?

Reply:
Agree, let's just remove it.